src/styles/blocks/card.scss

```diff
@@ -0,0 +1,84 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  max-width: 198px;
+  height: 406px;
+  background: #{$second-light-color};
+  border: 1px solid #{$first-light-color};
+  border-radius: 5px;
+
+  &__product-img {
+    width: 160px;
+    height: 134px;
+    padding: 32px 19px 0;
+  }
+
+  &__name-product {
+    width: 166px;
+    height: 36px;
```

Comment:
do not hardcode width and height of inner elements, use paddings for card and margins for vertical spacing between inner elements